docs/Using Notebooks.md/Working with Data.md

```diff
@@ -0,0 +1,72 @@
+# Connecting to Data
+Currently, you can connect to and query **Microsoft SQL Server Databases** or **Kusto Cluster Databases**. 
+
+To connect to data, you will need to reference Nuget packages and use the `#!connect` command. 
+
+## Microsoft SQL Server
+
+The syntax to reference the SQL Nuget Package: 
+
+>`#r "nuget: Microsoft.DotNet.Interactive.SqlServer, *-*"`
+
+The syntax to use the `#!connect` command: 
+
+> `#!connect` mssql `--kernel-name` <_connection-alias_> "<_connection-string_>"
+
+## Kusto Clusters
+
+The syntax to reference the KQL Nuget Package: 
+
+>`#r "nuget: Microsoft.DotNet.Interactive.Kql, *-*"`
+
+The syntax to use the `#!connect` command: 
+
+> `#!connect` kql `--kernel-name` <_connection-alias_> `--cluster` "<_connection-URI_>" `--database` "<_database-name_>"
+
+## Querying
+
+To connect to data and query in Polyglot Notebooks in VS Code, follow the steps below: 
+
+1. Create a new Polyglot Notebook starting with a C# cell. 
+
+1. In the first cell, use the required Nuget package from above. Make sure this has it's own cell and run it. 
+
+1. Add another C# cell and use the corresponding `#!connect` command from above to establish your connection and run the cell.
+    - The `--kernel-name` parameter is used to alias your data connection. This is the name you'd like to refer to this connection as while you continue to work in your notebook. It will become available in the dynamic kernel picker in the bottom right of cells. 
```

Comment:
nit nit: The `as` can be dropped - "refer to this connection ~~as~~ while you continue"

Reply:
I think it is needed...reads weird to me without it